packages/manager/modules/pci/src/projects/project/private-networks/delete/delete.routing.js

```diff
@@ -1,18 +1,20 @@
 export default /* @ngInject */ ($stateProvider) => {
   $stateProvider.state('pci.projects.project.privateNetwork.delete', {
-    url: '/delete?networkId',
+    url: '/delete',
```

Comment:
Here I need subnet as I need to display the name as well, which subnet user has deleted. Although not sure either subnet will have a name or not because I am not wrong subnet is being created internally while private network creation.